frontend/src/components/WikiList.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+class WikiList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleWikiChange = this.handleWikiChange.bind(this)
+    this.state = {error: false, wikis: []}
```

Comment:
Nitpick: you're storing `false` and `null` for errors here. Maybe just use `null`?